packages/network/src/api/image/requests.ts

```diff
@@ -96,6 +100,10 @@ export interface Add2ShotCloseUpImageOptions {
    * Boolean indicating if this picture is the first picture or the second picture of the 2-shot process.
    */
   firstShot: boolean;
+  /**
+   * Boolean indicating if a thumbnail request will be send when addImage is called.
+   */
+  enableThumbnail: boolean;
```

Comment:
_...will be sen**t** when..._

Also : rename variable to `useThumbnailCaching`, set it to optional with default : `true`